data/faq/is_mentor_necessary.md

```diff
@@ -0,0 +1,5 @@
+No! You just have to choose a project and write an awesome proposal. Mentors are set after we choose the projects (which is done by choosing the students).
+
+Our policy is that we'll always mentor awesome projects, so as long as your proposal is one of the wining ones, there will be a mentor for the project. But as for all other projects it might help to talk to your possible mentors, and in that case it also means trying to maybe find a mentor or at least talk to devs/maintainers about your ideas.
```

Comment:
This answers to the special case of the project without mentors on the projects page so you have to ask the right question or add the context in the answer :)